### ISSUE TITLE
cannot specify PHP version

### ISSUE THREAD

**Issue by individual-it** [owncloud-ci/core#3]
owncloud 10.1 (current master) requires PHP 7.1 but for this plugin there is no way to specify the PHP version, it always uses PHP 7.0
So installing master does not work
```
$ wget -qO- https://download.owncloud.org/community/owncloud-daily-master-qa.tar.bz2 | tar -xj -C /var/www/owncloud --strip 1
installing owncloud
$ wait-for-it mysql:3306
wait-for-it: waiting 15 seconds for mysql:3306
wait-for-it: mysql:3306 is available after 0 seconds
/var/www/owncloud /var/www/owncloud/apps/user_ldap
$ php ./occ maintenance:install -vvv         --database=mysql         --database-name=owncloud         --database-table-prefix=oc_         --admin-user=admin         --admin-pass=admin         --data-dir=/var/www/owncloud/data --database-host=mysql:3306                    --database-user=owncloud                    --database-pass=owncloud
This version of ownCloud requires at least PHP 7.1.0
You are currently running PHP 7.0.22-0ubuntu0.16.04.1. Please update your PHP version.
```

**Comment by patrickjahns** [comment]
I've bumped the php version to 7.1 in #4 

Not sure if it makes sense to also have different php versions inside the plugin - right now 7.1 is the common supported denominator between 10 and 10.1

**Comment by individual-it** [comment]
should be ok running on 7.1 its only the installation. Afterwards we can run the server with any PHP version by using apache on `owncloudci/php`

**Comment by tboerger** [comment]
Solved via #4
